packages/font-icons-mdl2/src/index.ts

```diff
@@ -22,7 +22,17 @@ import { IIconOptions } from '@fluentui/style-utilities';
 import { registerIconAliases } from './iconAliases';
 const DEFAULT_BASE_URL = 'https://spoppe-b.azureedge.net/files/fabric-cdn-prod_20210407.001/assets/icons/';
 
-export function initializeIcons(baseUrl: string = DEFAULT_BASE_URL, options?: IIconOptions): void {
+declare global {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  interface Window {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    FabricConfig: any;
+  }
+}
+export function initializeIcons(
+  baseUrl: string = window.FabricConfig?.fontBaseUrl ?? DEFAULT_BASE_URL,
```

Comment:
You should probably do:

```suggestion
  baseUrl: string = window?.FabricConfig?.fontBaseUrl ?? DEFAULT_BASE_URL,
```

Reply:
`const win = getWindow()` from utilities. If you search the repo, you'll find almost no usage of `window`

Reply:
Yeah, that's another one you could do

```suggestion
  baseUrl: string = getWindow().FabricConfig?.fontBaseUrl ?? DEFAULT_BASE_URL,
```

Reply:
I'm actually getting a build error using window, since it doesn't exist in a server rendering scenario, so I will change to `useWindow` hook